pennylane/ops/op_math/exp.py

```diff
@@ -179,7 +179,7 @@ def has_decomposition(self):
                     "when the base operator is a Tensor object with overlapping wires. "
                     f"Received base {self.base}."
                 )
-        return math.real(self.coeff) == 0 and is_pauli_word(self.base)
+        return math.real(self.coeff) == 0 and qml.pauli.is_pauli_word(self.base)
```

Comment:
Was the import causing problems?

Reply:
It was causing a circular import errors due to changes which are now split into a different PR. So undo-ing the import here